src/generator/AutoRest.Ruby/Model/MethodRb.cs

```diff
@@ -495,14 +495,7 @@ public virtual string ResponseGeneration()
         {
             IndentedStringBuilder builder = new IndentedStringBuilder("");
             builder.AppendLine("response = {0}_async({1}).value!", Name, MethodParameterInvocation);
-            if (ReturnType.Body != null)
-            {
-                builder.AppendLine("response.body unless response.nil?");
-            }
-            else
-            {
-                builder.AppendLine("nil");
-            }
+            builder.AppendLine("response.body unless response.nil?");
```

Comment:
@sarangan12 in the previous code, if body wasn't null, then we would return response.body (unless response was null), otherwise we'd return nil, since body was null. With the updated code I see that we're updating the resulting source code, does it change the resulting value? if response.body is null, what do we return? and if it's not, it looks like we're returning the same thing as before. What am I missing? Is there an example that would help see this in action?